MAUI/Chips/Customization.md

```diff
@@ -1180,6 +1222,48 @@ public MainPage()
 
 N> The default value of SelectedChipTextColor is [`Color.White`].
 
+## SelectedChipBackground
+
+The `SelectedChipBackground` property customizes the text color of the selected chip.
```

Comment:
The `SelectedChipTextColor` property customizes the text color of the selected chips.

Reply:
Changed